.github/workflows/job_build_windows.yml

```diff
@@ -180,9 +180,9 @@ jobs:
 
       - name: Cmake install - OpenVINO
         run: |
-          cmake --install . --config ${{ env.CMAKE_BUILD_TYPE }} --prefix ${{ env.INSTALL_DIR }}
-          cmake --install . --config ${{ env.CMAKE_BUILD_TYPE }} --prefix ${{ env.INSTALL_WHEELS_DIR }} --component python_wheels
-          cmake --install . --config ${{ env.CMAKE_BUILD_TYPE }} --prefix ${{ env.INSTALL_TEST_DIR }} --component tests
+          cmake --install . --prefix ${{ env.INSTALL_DIR }}
+          cmake --install . --prefix ${{ env.INSTALL_WHEELS_DIR }} --component python_wheels
+          cmake --install . --prefix ${{ env.INSTALL_TEST_DIR }} --component tests
```

Comment:
why? 
I would drop `-DCMAKE_BUILD_TYPE=${{ env.CMAKE_BUILD_TYPE }} `` above and keep build type here

Reply:
hat was just a revert commit to test that exactly those changes introduced the regression. But the fix should be in configuration step, you are right. We build release type now instead of debug (